packages/monaco/src/browser/monaco-editor-provider.ts

```diff
@@ -86,6 +87,7 @@ export class MonacoEditorProvider {
         @inject(MonacoWorkspace) protected readonly workspace: MonacoWorkspace,
         @inject(MonacoCommandServiceFactory) protected readonly commandServiceFactory: MonacoCommandServiceFactory,
         @inject(EditorPreferences) protected readonly editorPreferences: EditorPreferences,
+        @inject(FileSystemPreferences) protected readonly filePreferences: FileSystemPreferences,
```

Comment:
Please make this new param last and optional (not inversify's optional tho). Or is it possible to extract this into an injected property with the new logic in `postConstruct`?

This is to avoid breaking API.

Reply:
Yes, good point. I was blindly following the existing style. I've moved it to be injected as a property, instead.

Reply:
I just noticed that the new logic was in a method too which means it is trivial to inject as a a prop, nice!